certified-connectors/Experlogix Smart Flows/apiDefinition.swagger.json

```diff
@@ -0,0 +1,413 @@
+                    "Flows"
+                ]
+            }
+        },
+        "/api/DownloadDocument": {
+            "post": {
+                "consumes": [
+                    "application/json"
+                ],
+                "description": "Downloads the document from an execution.",
+                "operationId": "DownloadDocument",
+                "parameters": [
+                    {
+                        "in": "body",
+                        "name": "req",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/DownloadDocumentRequest"
+                        }
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Success",
+                        "schema": {
+                            "format": "binary",
+                            "title": "Document Content",
+                            "type": "string"
+                        }
+                    },
+                    "400": {
+                        "description": "Bad Request"
+                    },
+                    "401": {
+                        "description": "Unauthorized"
+                    }
+                },
+                "summary": "Download Document",
+                "tags": [
+                    "Documents"
+                ]
+            }
+        },
+        "/api/ExecuteFlow": {
+            "post": {
+                "consumes": [
+                    "application/json"
+                ],
+                "description": "Execute a Flow.",
+                "operationId": "ExecuteFlow",
+                "parameters": [
+                    {
+                        "in": "body",
+                        "name": "req",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/ExecuteFlowRequest"
+                        }
+                    }
+                ],
+                "produces": [
+                    "application/json"
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Success",
+                        "schema": {
+                            "$ref": "#/definitions/FlowExecutionResponse"
+                        }
+                    },
+                    "400": {
+                        "description": "Bad Request"
+                    },
+                    "401": {
+                        "description": "Unauthorized"
+                    }
+                },
+                "summary": "Execute Flow",
+                "tags": [
+                    "Flows"
+                ]
+            }
+        },
+        "/api/Flows": {
+            "get": {
+                "consumes": [
+                    "application/json"
+                ],
+                "operationId": "GetFlows",
```

Comment:
@lpperras all operations should have a description. Please add to all operations. for more information plese see our docs [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations).

Reply:
Added it. Didn't know it was now required with the x-ms-visibility property set to internal.